src/main/java/com/microsoft/azure/functions/worker/handler/WorkerStatusRequestHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.microsoft.azure.functions.worker.handler;
+import com.microsoft.azure.functions.rpc.messages.*;
+
+public class WorkerStatusRequestHandler extends MessageHandler<WorkerStatusRequest, WorkerStatusResponse.Builder> {
+
+    public WorkerStatusRequestHandler() {
+        super(StreamingMessage::getWorkerStatusRequest,
+              WorkerStatusResponse::newBuilder,
+              null,
+              StreamingMessage.Builder::setWorkerStatusResponse);
+    }
+
+    @Override
+    String execute(WorkerStatusRequest request, WorkerStatusResponse.Builder response) {
+        return "";
```

Comment:
Can we write a completed message like the other handlers do, e.g. `return String.format("WorkerStatusRequest completed");`?